lab-kyle/scss/base.scss

```diff
@@ -0,0 +1,9 @@
+// import librarys
+@import "vendor/normalize.css";
+// import partials
+@import "_theme";
+// write base styles
+
+body {
+  background-color: #ff7e7a;
```

Comment:
The point of this was to import colors from _theme.scss. It's setup to work, so just change this to $app-pink or $app-blue.